src/test/kotlin/org/rust/ide/refactoring/implementMembers/ImplementMembersHandlerTest.kt

```diff
@@ -29,14 +29,14 @@ class ImplementMembersHandlerTest : RsTestBase() {
 
     private fun invokeVia(actionInvoker: () -> Unit) {
         checkByText("""
-            trait T { fn f1(); }
+            trait T { async unsafe fn f1(); }
```

Comment:
Maybe it's better to create separate test for this case?

Reply:
Fixed.